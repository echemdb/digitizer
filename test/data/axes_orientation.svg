<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:cc="http://creativecommons.org/ns#"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
   xmlns:svg="http://www.w3.org/2000/svg"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:xlink="http://www.w3.org/1999/xlink"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   height="6820px"
   version="1.1"
   width="5212px"
   id="svg53"
   sodipodi:docname="axes_orientation.svg"
   inkscape:version="0.92.2 (5c3e80d, 2017-08-06)">
  <metadata
     id="metadata28">
    <rdf:RDF>
      <cc:Work
         rdf:about="">
        <dc:format>image/svg+xml</dc:format>
        <dc:type
           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
      </cc:Work>
    </rdf:RDF>
  </metadata>
  <sodipodi:namedview
     id="namedview55"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="0.19208211"
     inkscape:cx="2029.5638"
     inkscape:cy="5399.379"
     inkscape:window-width="1920"
     inkscape:window-height="1027"
     inkscape:window-x="-8"
     inkscape:window-y="22"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg53" />
  <defs
     id="defs49" />
  <image
     sodipodi:absref="E:\Github\echemdb\svgdigitizer\test\data\mello_2018_bromide_18562_p1.png"
     xlink:href="mello_2018_bromide_18562_p1.png"
     height="6820px"
     sodipodi:insensitive="true"
     width="5212px"
     x="0"
     y="0"
     id="image51" />
  <text
     xml:space="preserve"
     style="font-size:85.3333px;line-height:1.25;font-family:sans-serif"
     x="1197.1061"
     y="1465.9803"
     id="text4145"><tspan
       sodipodi:role="line"
       id="tspan4143"
       x="1197.1061"
       y="1465.9803">scan rate: 50 mV / s</tspan></text>
  <g
     id="g33708">
    <text
       xml:space="preserve"
       style="font-size:85.3333px;line-height:1.25;font-family:sans-serif"
       x="1105.1626"
       y="950.65417"
       id="text2755"><tspan
         sodipodi:role="line"
         id="tspan2753"
         x="1105.1626"
         y="950.65417">curve: Pt(111)_ph_1,2_KPF6</tspan></text>
    <path
       style="fill:none;stroke:#000000;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
       d="m 856.0944,1261.0445 0.91671,-61.6643 0.91671,-30.3622 11.05443,-16.5636 21.13824,-3.6778 21.19217,-2.7652 15.63797,-0.9262 7.3876,-2.7516 24.80509,0.9127 24.85899,0.9262 35.91349,-0.9262 h 38.6635 l 26.6924,-0.9127 29.4426,0.9127 12.8878,-0.9127 18.3882,9.1945 23.9422,6.4429 31.276,7.3692 30.3593,8.2819 35.9134,5.5167 44.1639,2.7651 h 31.2759 l 52.4681,-0.9262 h 54.3016 l 40.497,-0.9127 36.7762,-4.604 34.08,-6.443 32.1927,-7.3692 24.859,-2.7515 31.2759,-4.6041 h 24.8591 l 28.5258,-2.7651 18.3881,-3.6778 20.2755,-8.2818 9.221,-13.8122 6.417,-19.3152 2.75,-15.6511 5.5543,11.047 3.6668,18.4026 8.2504,23.9329 12.8878,15.6374 16.6086,12.8859 15.638,1.8389 v 16.5637 l -10.1377,7.3692 -20.2215,7.3556 -13.8046,11.047 -9.221,16.5637 -5.5542,19.3288 -3.6669,13.7985 -1.8333,17.49 -5.5003,-8.2819 -1.8334,-23.9329 -6.4709,-13.7985 -16.5547,-9.2081 -25.7757,-3.6778 -34.9967,-2.7651 -26.6385,-3.6778 -45.1345,-6.443 -53.3848,-10.1207 -42.3304,-5.5167 -39.5264,-0.9263 h -41.4137 -43.301 -40.4431 -41.4137 l -37.7468,3.6778 -42.3305,8.2819 -23.9422,6.4429 -59.8018,11.047 -16.5547,5.5303 -42.3844,-0.9262 -28.5258,3.6778 -45.08054,0.9262 -38.66357,0.9127 -37.69291,1.8389 -29.44255,-0.9127 -17.52533,4.6041 -10.0838,11.0333 z"
       id="path33623" />
  </g>
  <text
     xml:space="preserve"
     style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:80px;line-height:1.25;font-family:sans-serif;-inkscape-font-specification:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none"
     x="249.89313"
     y="312.36636"
     id="text837"><tspan
       sodipodi:role="line"
       id="tspan835"
       x="249.89313"
       y="312.36636">comment: Working solutions of the pH-series with values lower than 3 were prepared by using HClO4 and KClO4. For pH values ranging from 3 to 9, phosphate buffer solutions were employed (Na2HPO4 and NaH2PO4). Alkaline solutions were prepared with NaOH and KClO4. The exact concentrations of these components are unkown.</tspan></text>
  <g
     id="g1380">
    <path
       style="fill:none;stroke:#000000;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
       d="M 533.31288,1333.943 782.5011,1546.586"
       id="path4645" />
    <text
       xml:space="preserve"
       style="font-size:53.3333px;line-height:1.25;font-family:sans-serif;-inkscape-font-specification:sans-serif;letter-spacing:0px;word-spacing:0px"
       x="105.61523"
       y="1350.5088"
       id="text4126"><tspan
         sodipodi:role="line"
         id="tspan4124"
         x="105.61523"
         y="1350.5088">j1: -250 uA / cm2</tspan></text>
  </g>
  <g
     id="g6260">
    <path
       style="fill:none;stroke:#000000;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
       d="m 906.00013,1616.4919 113.71267,305.1329"
       id="path6095" />
    <text
       xml:space="preserve"
       style="font-size:53.3333px;line-height:1.25;font-family:sans-serif;-inkscape-font-specification:sans-serif;letter-spacing:0px;word-spacing:0px"
       x="1016.9518"
       y="1934.5093"
       id="text5628"><tspan
         sodipodi:role="line"
         id="tspan5626"
         x="1016.9518"
         y="1934.5093">E1: 0.1 V vs. RHE</tspan></text>
  </g>
  <g
     id="g8274">
    <path
       style="fill:none;stroke:#000000;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
       d="M 2062.4335,1815.7882 1883.3202,1614.6666"
       id="path8109" />
    <text
       xml:space="preserve"
       style="font-size:53.3333px;line-height:1.25;font-family:sans-serif;-inkscape-font-specification:sans-serif;letter-spacing:0px;word-spacing:0px"
       x="2059.6726"
       y="1834.1946"
       id="text7518"><tspan
         sodipodi:role="line"
         id="tspan7516"
         x="2059.6726"
         y="1834.1946">E2: 0.9 V vs. RHE</tspan></text>
  </g>
  <g
     id="g1861">
    <path
       style="fill:none;stroke:#000000;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
       d="M 786.12215,790.02673 499.78627,658.57253"
       id="path1696" />
    <text
       xml:space="preserve"
       style="font-size:53.3333px;line-height:1.25;font-family:sans-serif;-inkscape-font-specification:sans-serif;letter-spacing:0px;word-spacing:0px"
       x="96.073799"
       y="640.7901"
       id="text1728"><tspan
         sodipodi:role="line"
         id="tspan1726"
         x="96.073799"
         y="640.7901">j2: 300 uA / cm2</tspan></text>
  </g>
  <text
     xml:space="preserve"
     style="font-style:normal;font-weight:normal;font-size:40px;line-height:1.25;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none"
     x="1249.4657"
     y="562.25952"
     id="text838"><tspan
       sodipodi:role="line"
       id="tspan836"
       x="1249.4657"
       y="562.25952">figure: 1a</tspan></text>
</svg>
