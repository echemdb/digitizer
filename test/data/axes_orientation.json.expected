{"resources": [{"path": "axes_orientation.csv", "name": "axes_orientation", "profile": "tabular-data-resource", "scheme": "file", "format": "csv", "hashing": "md5", "encoding": "utf-8", "schema": {"fields": [{"name": "t", "unit": "s", "type": "number"}, {"name": "E", "unit": "V", "reference": "RHE", "type": "number"}, {"name": "j", "unit": "A / m2", "type": "number"}]}}], "experimental": {"tags": []}, "source": {"figure": "1a", "curve": "Pt(111)_ph_1,2_KPF6"}, "figure description": {"version": 1, "type": "digitized", "simultaneous measurements": [], "measurement type": "CV", "scan rate": {"value": 50.0, "unit": "mV / s"}, "fields": [{"name": "E", "unit": "V", "orientation": "x", "reference": "RHE"}, {"name": "j", "unit": "uA / cm2", "orientation": "y"}], "comment": "Working solutions of the pH-series with values lower than 3 were prepared by using HClO4 and KClO4. For pH values ranging from 3 to 9, phosphate buffer solutions were employed (Na2HPO4 and NaH2PO4). Alkaline solutions were prepared with NaOH and KClO4. The exact concentrations of these components are unkown."}, "data description": {"version": 1, "type": "digitized", "measurement type": "CV"}, "profile": "data-package"}